src/docfx/lib/json/JsonUtility.cs

```diff
@@ -259,9 +259,9 @@ public static void Merge(JObject container, JObject overwrite)
 
             JToken DeepClone(JToken token)
             {
+                var lineInfo = token as IJsonLineInfo;
```

Comment:
use `(IJsonLineInfo)token)` here over `as` because `as` return a `null` object when the conversion failed. `(IJsonLineInfo)token)` throws when the cast failed.